bridge/qb/shared/main.lua

```diff
@@ -1,5 +1,21 @@
+local starterItems = require 'config.server'.starterItems
 local qbShared = require 'shared.main'
 
+---@deprecated use starterItems in config/server.lua
+qbShared.StarterItems = function()
```

Comment:
Is this supposed to be a function, or a table?

Reply:
its a table. my mistake